src/main/kotlin/com/team4099/robot2021/commands/climber/LockClimber.kt

```diff
@@ -10,6 +11,9 @@ class LockClimber : CommandBase() {
 
   override fun initialize() {
     Climber.setOpenLoopPower(0.0)
-    Climber.brakeApplied = false
+    Climber.brakeApplied = true
+    Logger.addEvent("LockClimber", "Climber Locked")
```

Comment:
make the first parameter for all of these "Climber", its meant to refer to the subsystem

(not sure if someone else did it incorrectly)